libs/ui-lib/lib/ocm/components/clusterConfiguration/manifestsConfiguration/components/CustomManifestsForm.tsx

```diff
@@ -39,7 +39,9 @@ const AutosaveWithParentUpdate = ({
   const isAutoSaveRunning = useFormikAutoSave();
 
   React.useEffect(() => {
-    onFormStateChange({ isSubmitting, isAutoSaveRunning, isValid, touched, errors, isEmpty });
+    setTimeout(() => {
+      onFormStateChange({ isSubmitting, isAutoSaveRunning, isValid, touched, errors, isEmpty });
+    }, 100);
```

Comment:
```suggestion
    }, 100 // Why100? Why not 99? MAGIC NUMBERS...);
```

Reply:
I think with the other changes I don't need this timeout. I'll check it